source/vir/src/func_to_air.rs

```diff
@@ -369,6 +369,9 @@ pub fn func_decl_to_air(
                 let axiom = Arc::new(DeclX::Axiom(expr));
                 decl_commands.push(Arc::new(CommandX::Global(axiom)));
             }
+            if function.x.attrs.bit_vector {
+                // TODO: function level bitvector mode
```

Comment:
Should we have an `unimplemented!()` here?

Reply:
Is this code path reachable? If so, and this is a placeholder for an upcoming feature, I'd much rather return error instead of crashing, with something like:
```
pub(crate) fn unsupported_err_span<A>(span: Span, msg: String) -> Result<A, VirErr> {
```
( https://github.com/secure-foundations/verus/blob/98d4c5bee9db2dfbe7e1a02bafac4258f51bdd28/source/rust_verify/src/util.rs#L34-L39 )

which would need to be moved/replicated in this crate.

Reply:
I think this code is only reachable when `#[verifier(bit_vector)]` attribute is specified. Currently we don't have test cases developed for that just yet.